cmd/satellite/open_stream.go

```diff
@@ -61,13 +61,17 @@ func NewOpenStreamCommand() *cobra.Command {
 			if err := flags.ValidateAll(); err != nil {
 				return err
 			}
-
+			_, err := time.Parse("2006-01-02'T'15:04:05Z", openStreamFlag.AutoCloseTime)
```

Comment:
Is the format supposed to have single quotes around the T?

`2006-01-02'T'15:04:05Z` -> `2006-01-02T15:04:05Z`

Standard is described here: https://en.wikipedia.org/wiki/ISO_8601 and gives `2020-10-16T04:37:44Z` as an example.

Reply:
Ahh, yeah you're right.

Reply:
Could the Help Text also clarify the delay param - is it just a number of seconds? (what max..?)
Some comments above suggest something like 5s rather than just 5